src/P/Functor.hs

```diff
@@ -8,5 +8,6 @@ import           Data.Function (flip)
 import           Data.Functor as X (Functor(..), ($>), (<$>), void)
 
 with :: Functor f => f a -> (a -> b) -> f b
-with =
-  flip fmap
+with xs f =
+  fmap f xs
+{-# INLINE with #-}
```

Comment:
`INLINABLE` would be ok, but I can't think of a reason why you wouldn't always want to inline this as it is just a synonym for `fmap`, so inlining will never increase code size